bitmap/bitmap_test.go

```diff
@@ -0,0 +1,129 @@
+package bitmap
+
+import (
+	"testing"
+
+	. "gopkg.in/check.v1"
+)
+
+func Test(t *testing.T) { TestingT(t) }
+
+type TestSuite struct{}
+
+var _ = Suite(&TestSuite{})
+
+func (s *TestSuite) TestBitmap(c *C) {
+	bm, err := NewBitmap(200, 100)
+	c.Assert(bm, IsNil)
+	c.Assert(err, NotNil)
+
+	bm, err = NewBitmap(100, 200)
+	c.Assert(err, IsNil)
+
+	_, _, err = bm.AllocateRange(0)
+	c.Assert(err, NotNil)
+
+	start, end, err := bm.AllocateRange(100)
+	c.Assert(err, IsNil)
+	c.Assert(start, Equals, int32(100))
+	c.Assert(end, Equals, int32(199))
+
+	start, end, err = bm.AllocateRange(1)
+	c.Assert(err, IsNil)
+	c.Assert(start, Equals, int32(200))
+	c.Assert(end, Equals, int32(200))
```

Comment:
For `bm, err = NewBitmap(100, 200)`, is port `200` included?

Reply:
Yes, it is.